src/proof/nova.rs

```diff
@@ -223,6 +224,28 @@ pub fn circuits<'a, F: CurveCycleEquipped, C: Coprocessor<F> + 'a>(
     )
 }
 
+/// For debugging purposes, synthesize the circuit and check that the constraint
+/// system is satisfied
+pub(crate) fn debug_step<F: LurkField, C: Coprocessor<F>>(
```

Comment:
Consider `#[inline]`

Reply:
I can add it but it shouldn't be the cause of the regression, right? It's not being called